prac_05/state_names.py

```diff
@@ -4,15 +4,18 @@
 File needs reformatting
 """
 
+
 # TODO: Reformat this file so the dictionary code follows PEP 8 convention
 STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
-# print(STATE_NAMES)
 
-state = input("Enter short state: ")
+state = input("Enter short state: ").upper()
 while state != "":
     if state in STATE_NAMES:
         print(state, "is", STATE_NAMES[state])
     else:
         print("Invalid short state")
-    state = input("Enter short state: ")
+    state = input("Enter short state: ").upper()
+
+for state, value in STATE_NAMES.items():
+    print("{:<4} is {}".format(state, value))
```

Comment:
Should have newline at end of file for PEP8

Reply:
Done.